etc/db_schema_main.sql

```diff
@@ -26,3 +26,12 @@ SOURCE etc/db_schema_user_role_table.sql;
 
 /* #4: Create User session table */
 SOURCE etc/db_schema_user_session_table.sql;
+
+/* #5: Create University table */
```

Comment:
The "#number" isn't super important, but you should either correct the sequence or numbers or take them all out.